.eslintrc.js

```diff
@@ -14,5 +15,7 @@ module.exports = {
   rules: {
     "react/jsx-props-no-spreading": "warn",
     "react/require-default-props": "off",
+    "react/function-component-definition": "off",
```

Comment:
Consider using the default rule (not overwrite it) and see if it adds any value to having a consistent code style.

If having the default rule helps everyone on a team to use the same convention, then this is a good candidate for a keeper.

Reply:
Good catch, will update it to allow only for arrow function components as that is the preferred way of writing for most people 

```
    "react/function-component-definition": [
      2,
      { namedComponents: "arrow-function" },
    ],
 ```